### DISCUSSION TITLE
TTS multi-speaker & multi-language, support for low resource languages

### DISCUSSION THREAD

**Discussion by showgan** [speechbrain/speechbrain#2179] [Feature Request]
Hi,

Are there any plans to support training a mutli-speaker TTS model? or is it supported already (if so then which recipe to use)?

Similar question about creating a multi-language model.

Related to this, having the above capabilities can help with better support for low resource languages by utilizing fine-tuning on a multi-language and multi-speaker pre-trained model.
I suppose that this is relevant for vocoders as well.

Thanks!

**Comment by mravanelli** [comment]
Hi,
thank you for asking.
Yes, we are finalizing a PR for zero-shot multi-speaker TTS:
https://github.com/speechbrain/speechbrain/pull/2120.
Feel free to take a look. At the moment, we are working to improve the
quality as much as we can. Multilingual TTS is another very interesting
topic, but we do not have bandwidth at the moment for that, unless someone
in the community would like to contribute to that.

**Reply by showgan** [reply]
Thanks for the reply!

A follow-up question: How about training for another language. Not training a multi-lingual model,  but training a model for a single, non-English, language - specifically, a language which is not supported by known phonemizers and tokenizer.
I have my own implementation of graphemem-to-phoneme for this language.

It will be very helpful if there were some documentation on this, e.g. how to train or replace the internal g2p and tokenizer modules, changes needed in various yaml files such as the "lexicon" parameter in recipes/LJSpeech/TTS/fastspeech2/hparams/train_internal_alignment.yaml etc.
